src/devtools/devtools.tsx

```diff
@@ -133,6 +133,33 @@ export function ReactQueryDevtools({
     setIsResolvedOpen(isOpen)
   }, [isOpen, isResolvedOpen, setIsResolvedOpen])
 
+  const handlePanelTransitionStart = React.useCallback(() => {
+    if (panelRef.current && isResolvedOpen) {
+      panelRef.current.style.visibility = 'visible'
+    }
+  }, [isResolvedOpen])
+
+  const handlePanelTransitionEnd = React.useCallback(() => {
+    if (panelRef.current && !isResolvedOpen) {
+      panelRef.current.style.visibility = 'hidden'
+    }
+  }, [isResolvedOpen])
```

Comment:
If we move those function definitions _into_ the `useEffect` where they are used, we don't need to wrap them in `useCallback. What do you think?

Reply:
Sure, I can do that.